tests/js/spec/components/projectHeader/projectSelector.spec.jsx

```diff
@@ -108,18 +108,6 @@ describe('ProjectSelector', function() {
       expect(wrapper.find('.dropdown-menu').length).toBe(1);
     });
 
-    it('closes dropdown when project is selected', function() {
```

Comment:
would be down to help figure out how to keep this test, logically it /should/ work

Reply:
👍

Reply:
This seems related to the above test case that was removed?